platform/feature-api/communication/remote-pod/src/async.ts

```diff
@@ -104,16 +104,24 @@ class AsyncInfo implements Info {
     }
 }
 
-class AsyncNetwork implements Network {
-    constructor(private readonly promise: Promise<Network>) {}
+class AsyncEndpoint implements Endpoint {
+    constructor(private readonly promise: Promise<Endpoint>) {}
 
-    async httpPost(
-        url: string,
-        body: string,
+    async send(
+        endpointId: string,
+        payload: string,
         contentType?: string,
         authorization?: string
-    ): Promise<string | undefined> {
-        return (await this.promise).httpPost(url, body, contentType, authorization);
+    ): Promise<void> {
+        return (await this.promise).send(endpointId, payload, contentType, authorization);
```

Comment:
OK, let me see if I get this. This is an async endpoint, so it's going to request the endpoint value *every single time it's invoked*. Makes sense because it's async, So this would allow the endpoint to be changed in runtime, is that correct?
Anyway, my understanding is that this is only used for testing, is that correct?